iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -994,6 +994,15 @@ cluster_name=defaultCluster
 # The sync regular interval (in minutes) for the PipeMetaSyncer service.
 # pipe_meta_syncer_sync_interval_minutes=3
 
+# Whether to enable pipe auto restart after pipe meets error during runtime.
+# pipe_exception_stopped_auto_restart_enabled=true
+
+# When the pipe meta synchronization round is a multiple of it, it will restart pipes stopped automatically by critical exceptions.
+# Useless if the pipe_exception_stopped_auto_restart_enabled is set to false.
+# Eg: if pipe_meta_syncer_sync_interval_minutes=3, and pipe_meta_syncer_auto_restart_pipe_round=5,
+# then the pipe restart process's interval will be 15 minutes, and is executed by the pipe meta sync thread.
+# pipe_meta_syncer_auto_restart_pipe_round=5
```

Comment:
Minor code style issues: it's best to put together parameters (methods) with the same prefix, which makes the code cleaner